numberformat/diff.emu

```diff
@@ -1018,7 +1018,10 @@
           1. Else,
             1. Let _str_ be ! Number::toString(_x_).
         1. If the grammar cannot interpret _str_ as an expansion of |StringNumericLiteral|, return *NaN*.
```

Comment:
Should we delete line 1020 now that we check for NaN based on the "List of errors" returned from ParseText?

Reply:
LGTM